test/config_test.py

```diff
@@ -39,8 +39,10 @@ def gen_new_config(config_file, training_service='local'):
     new_config_file = config_file + '.tmp'
 
     ts = get_yml_content('training_service.yml')[training_service]
-    print(config)
-    print(ts)
+    if training_service == 'remote':
+        ts['machineList'][0]['port'] = int(os.environ['docker_port'])
+        ts['machineList'][0]['username'] = os.environ['docker_user']
```

Comment:
Please do not use environment variables here,  write remote machine configuration into training_service.yml,  then call this config_test.py